modloader/modloader.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/tsaikd/gogstash/filter/ratelimit"
 	"github.com/tsaikd/gogstash/filter/removefield"
 	"github.com/tsaikd/gogstash/filter/typeconv"
+	"github.com/tsaikd/gogstash/filter/url_param"
```

Comment:
Please change package path from `url_param` to `urlparam` for consistency.

Reply:
Package path has been modified